app/scripts/modules/amazon/src/loadBalancer/configure/classic/AdvancedSettings.tsx

```diff
@@ -23,44 +23,43 @@ class AdvancedSettingsImpl extends React.Component<IAdvancedSettingsProps> {
         <FormikFormField
           name="healthTimeout"
           label="Timeout"
+          required={true}
+          fastField={false} /* This field depends on  healthInterval */
           help={<HelpField id="loadBalancer.advancedSettings.healthTimeout" />}
-          input={NumberInput}
-          validate={[
-            Validation.minValue(0),
-            Validation.maxValue(values.healthInterval, 'Timeout must be less than the health Interval.'),
-          ]}
+          input={props => <NumberInput {...props} min={0} max={values.healthInterval} />}
+          validate={Validation.maxValue(values.healthInterval, 'Timeout must be less than the health Interval.')}
```

Comment:
```suggestion
          validate={Validation.maxValue(values.healthInterval, 'Timeout must be less than the health interval.')}
```

I know you just copy/pasted my bad capitalization, but might as well fix it here ;)